### ISSUE TITLE
backend: set new password

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#43]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* password received from frontend is checked with respect to the password rules: 8 elements. Must contain digits, letters, and special characters
* if valid password, store it in the database and send success message to frontend
* if invalid password, send error message to frontend

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
